app/src/main/java/com/beemdevelopment/aegis/ui/PreferencesFragment.java

```diff
@@ -206,6 +206,12 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
         });
         timeoutPreference.getOnPreferenceChangeListener().onPreferenceChange(timeoutPreference, timeoutPreference.getText());*/
 
+        Preference codeDigitGroupingPreference = findPreference("pref_code_grouping");
```

Comment:
The app crashes on this line for me. Shouldn't this be ``pref_code_group_size``?

Reply:
Yes, thank you. Decided last minute to rename it 🙈